### PR TITLE
Us7 t3 condo requests view

### PR DESCRIPTION
**Pull request by mattflahertyy**
Only one default request component is shown for each condo. 

BACKEND to implement :
- iterate through each requests PER condo and display these components (inside CondoDetails.jsx)
- add functionality to the CondoRequestsView.jsx to get request data and display
- add functionality to update backend request once company presses advance

*****ABOVE IS COMPLETED******

just missing tests and documentation but I'll do that in a different pr, i want to merge this asap to reduce conflicts

### CODE REVIEW

File: mini-cap/src/components/CondoRequestsView.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    const [currentStep, setCurrentStep] = useState(1);
+
+    useEffect(() => {
+    }, []);
+
+    const handleAdvance = () => {
+        if (currentStep < 4)
+            setCurrentStep(currentStep + 1);
+
+        // updateRequest(condoId, requestId);
```
Comment (leobrod44) [inline mini-cap/src/components/CondoRequestsView.jsx:16]: Go through the upateRequest method and see stepType. The return value while be the step its currently at

File: mini-cap/src/components/CondoRequestsView.jsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import { MANAGEMENT_COMPANY } from "../backend/Constants";
+
+const CondoRequestsView = ({ role }) => {
+    const [currentStep, setCurrentStep] = useState(1);
+
+    useEffect(() => {
+    }, []);
+
+    const handleAdvance = () => {
+        if (currentStep < 4)
```
Comment (leobrod44) [inline mini-cap/src/components/CondoRequestsView.jsx:13]: no need to check this, instead check if the updateRequest returns "Completed"

**Review by leobrod44 — CHANGES_REQUESTED** [review]

**Comment by leobrod44** [comment]
Also, change your commit messages to include the issue number at the end as per code management requirement

**Review by laurenrigante — COMMENTED** [review]
Ui looks good. 

- i would move the condo requests section ABOVE the edit and delete buttons on the condo details page, i feel like edit and delete should be the last things there
image

- i dont think we need a previous button for the requests, i dont think it really makes sense to have one. a mgmt company should only be able to move it forward i dont think they should be able to move it back

- this is a personal opinion, but i think we should change the blue color of  the circles to a blue that matches the rest of the website. The main blue we use is  #4540db , theres also a lighter lbue version we use for titles that  could be an option as well. Here ishow it looks with the blue of the website
image



- lastly, i think you can make the font size of the writing for request type and description smaller. In comparison to the rest of the condo details writing, its a much larger font.
- 
image

**Review by laurenrigante — APPROVED** [review]
UI updates look good.

**Review by leobrod44 — APPROVED** [review]

**Pull request merged**